scripts/pydev_register

```diff
@@ -17,7 +17,12 @@ fi
 if [ -z "${1}" ]; then
   usage
   exit
+elif [ ! -f "${1}" ]; then
```

Comment:
`-f` is specifically for files, did you mean `-d` for a directory check or a `-e` for a generic existence check? I think this is typically called on directories

Reply:
We pass the module *.py or bin file to be registered I thought. I was thinking we want to make sure these files exist?

Reply:
the path to these files. I'm unsure, do we check the dir exist or the file using the passed path.

Reply:
normally I'd do something like this (with my gh clone area in my home):
```
cd ~/github/pcdsdevices
pydev_register pcdsdevices
```

Reply:
"Register the path `pcdsdevices` as a `module` for my dev env"

Reply:
ah, so dont just register individual modules

Reply:
I think you could add a standalone .py file module, I haven't done that myself but it should work in those rare cases where you do only just have a one-file module

Reply:
okay, looks like we want to go with `-e` here to catch individual modules and dirs.